recipes-kernel/linux/linux-iot2050-6.1.inc

```diff
@@ -17,12 +17,36 @@ def get_patches(d, patchdir):
 
     return ' '.join(['file://' + patch[len(files_dir)+1:] for patch in patches])
 
+def get_extra_configs(d):
+    import re
+    import os
+
+    overrides = d.getVar('OVERRIDES')
+    append_config = []
+
+    if re.search("pvu-dma", overrides):
+        append_config.append('CONFIG_TI_PVU=y')
+
+    if append_config:
+        append_config_file = os.path.join(d.getVar('T'), 'iot2050_append_config.cfg')
+        append_config_dir = os.path.dirname(append_config_file)
+
+        if not os.path.exists(append_config_dir):
+            return ''
+        with open(append_config_file, 'w') as f:
+            for line in append_config:
+                f.write(line + '\n')
+        return 'file://' + append_config_file
+    else:
+        return ''
```

Comment:
No - just add a config snippet conditionally that enables the PVU.

Reply:
I plan to modify it to a template, also a patch to isar:

```py
def config_fragments(d):
    fragments = []
    sources = d.getVar("SRC_URI").split()
    for s in sources:
        _, _, local, _, _, parm = bb.fetch.decodeurl(s)
        apply = parm.get("apply")
        if apply == "no":
            continue
+        if local.endswith(".tmpl"):
+            local = local[:-5]
        base, ext = os.path.splitext(os.path.basename(local))
        if ext and ext in (".cfg"):
            fragments.append(local)
    return fragments
```

Reply:
As I wrote above, all this is unneeded, only adding complexity.

Reply:
ACK